tests/e2e/specs/graph.cy.js

```diff
@@ -69,4 +94,18 @@ describe('Graph View', () => {
       .should('have.length', 10)
       .should('be.visible')
   })
+
+  it('remembers autorefresh setting when switching between workflows', () => {
+    cy.visit('/#/workspace/one')
+    addView('Graph')
+    waitForGraphLayout()
+    checkRememberToolbarSettings('[data-cy=control-autoRefresh]', 'have.class', 'not.have.class')
+  })
+
+  it('remembers transpose setting when switching between workflows', () => {
+    cy.visit('/#/workspace/one')
+    addView('Graph')
+    waitForGraphLayout()
+    checkRememberToolbarSettings('[data-cy=control-transpose]', 'not.have.class', 'have.class')
+  })
 })
```

Comment:
I could add an extra test to check that the graph loads when switching between workflows

```
  it('loads graph when switching between workflows', () => {
    cy.visit('/#/workspace/one')
    addView('Graph')
    waitForGraphLayout()
    cy.visit('/#/workspace/other/multi/run2')
    addView('Graph')
    waitForGraphLayout()
    cy.visit('/#/workspace/one')
    cy
    // there should be 7 graph nodes (all on-screen)
      .get('.c-graph:first')
      .find('.graph-node-container')
      .should('be.visible')
      .should('have.length', 7)
  })
  ```

Reply:
Sounds good